clickhouse-client/src/main/java/com/clickhouse/client/config/ClickHouseClientOption.java

```diff
@@ -378,7 +378,13 @@ public enum ClickHouseClientOption implements ClickHouseOption {
      * false.
      */
     USE_TIME_ZONE("use_time_zone", "", "Time zone of all DateTime* values. "
-            + "Only used when use_server_time_zone is false. Empty value means client time zone.");
+            + "Only used when use_server_time_zone is false. Empty value means client time zone."),
+
+    /**
+     * Max connections of ApacheHttpConnectionImpl
+     */
+    APACHE_HTTP_CLIENT_MAX_CONNECTIONS("apache_http_client_max_connections", 10,
```

Comment:
Could you please move this option to `ClickHouseHttpClientOption`? We may promote it to a more generic one like `MAX_OPEN_CONNECTIONS` at a later stage.